### DISCUSSION TITLE
Not able to get past the macOS recovery screen

### DISCUSSION THREAD

**Discussion by Rick0038** [quickemu-project/quickemu#1222] [Quickemu]
Hi All,
I am experimenting with MacOS VM as i need one for my website testing. but i cant get past the recovery screen everytime i click on macos base system it goes through a bunch of text then goes back to recovery screen. KIndly help

Bases system is Ubuntu
Apps used quickemu_4.9.4-1_all and quickgui_1.2.8-1_jammy1.0_amd64

**Comment by Rick0038** [comment]
In the QEMU logs i can see this 
```
qemu-system-x86_64: warning: host doesn't support requested feature: CPUID.01H:ECX.pcid [bit 17]
qemu-system-x86_64: warning: host doesn't support requested feature: CPUID.07H:EBX.erms [bit 9]
qemu-system-x86_64: warning: host doesn't support requested feature: CPUID.07H:EBX.invpcid [bit 10]
```

**Comment by flexiondotorg** [comment]
The soon to be release Quickemu 4.9.5 offers significantly improved support for macOS VMs

**Reply by shalom938** [reply]
same issue, if i select macos base image it prints a lot of text and then crash, if select EFI it will do nothing for a while then come back to the same screen.
using opensuse tumbleweed with quickemu from source
